code/workspaces/infrastructure-api/src/stacks/jupyterStack.js

```diff
@@ -22,6 +23,7 @@ function createJupyterNotebook(params) {
   return secretManager.createStackCredentialSecret(name, type, projectKey, credentials)
     .then(createPySparkConfigMap(params))
     .then(createDaskConfigMap(params))
+    .then(createJupyterConfigMap(params))
```

Comment:
Think we want to delete it at the end as well if possible so it's not hanging around.

Reply:
Thanks, I'd missed that one!